packages/palette/src/elements/Select/Select.tsx

```diff
@@ -153,6 +153,10 @@ const resetMixin = css`
   }
 `
 
+interface CaretProps extends ExecutionContext {
+  disabled?: boolean
+}
+
 /** Creates a small caret */
 export const caretMixin = css`
```

Comment:
ditto here

Reply:
for some reason it doesn't like the mixin. @araujobarret - maybe you can take a look at these? Its being funny.